### PR TITLE
(2889) Tidy changelog

### PR DESCRIPTION
**Pull request by yndajas**
## Changes in this PR

- Fix changelog order (convention; dev experience)
- Tidy up (errors)
- Reformat headings and links (style)

You can view the updated changelog in GitHub style here: https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/tidy-changelog/CHANGELOG.md

### Before

image

[...]

image

### After

image

## Next steps

- [ ] Is an ADR required? An ADR should be added if this PR introduces a change to the architecture.
- [heh] Is a changelog entry required? An entry should always be made in `CHANGELOG.md`, unless this PR is a small tweak which has no impact outside the development team.
- [ ] Do any environment variables need amending or adding?
- [ ] Have any changes to the XML been checked with the IATI validator? See [XML Validation](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/doc/xml-validation.md)

### CODE REVIEW

**Comment by yndajas** [comment]
[Tidy changelog](https://trello.com/c/JWojnnRM/2889-tidy-changelog)

**Comment by yndajas** [comment]
> [...] not sure how I'm feeling about the titles though.
>
> I think the original creator probably got the title format from here https://keepachangelog.com/en/0.3.0/ (not the they are necessarily the arbiter of changelog formatting).

Yeah I think you're right, given that's linked in the Rails template changelog. I think even "[132](example.com) - 2023-03-04" would be cleaner than "[release-132](example.com) - 2023-03-04" (and still in line with the reference format). I probably slightly prefer the Rspec style, but it's a weakly-held view, and it might be best to stick to the format of that source (even if we deviate in some ways, e.g. not grouping our lists)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,660 +1,753 @@
...
+[Full changelog][unreleased]
+
+- Add budget revisions:
+  - Only budget value can be edited
+  - Add a budget revisions page
+  - Add comments when editing budgets to be shown alongside the revisions
+  - Improve design/content for the budgets area
+  - Show "original" and "revised" budgets on the XML exports
+
+## 132 / 2023-03-16
```
Comment (Gweaton) [inline CHANGELOG.md:14]: I'd prefer to see "Release" here, either as "release-132" or "Release 132" if we think that's neater. It _should_ be obvious what the numbers refer to here but that might not always be the case, particularly if this is a repository that's open to the public. I also find it a little easier to scan 👀

**Review by Gweaton — APPROVED** [review]
Thanks for tidying this up - have just added one comment re: titles

File: CHANGELOG.md (hunk 1 above)
Comment (yndajas) [reply CHANGELOG.md:14]: I did wonder about putting them all under a "Released" `h2` but that felt a little off. Maybe this?

Any thoughts @jkempster34? I wasn't sure if your comment on titles actually included the shifting of the changelog link actually, or just the format (Keep a Changelog includes the link in the heading)

---

## Release 132 - 2023-03-16

[Full changelog](example.com)

Comment (jkempster34) [reply CHANGELOG.md:14]: Um, honestly I just didn't like the slash because it looked funny to me!

I don't have any preferences beyond that

**Review by CristinaRO — APPROVED** [review]

**Pull request merged**